src/WebWorker.ts

```diff
@@ -76,14 +97,31 @@ export class WebWorker extends EventEmitter {
     );
   }
 
+  /**
+   * Returns the URL for the script this web worker is running.
```

Comment:
I think in this case we could lose the description in favour of just the `@returns ...` line rather than duplicate them both